cmd/cdi-controller/controller.go

```diff
@@ -288,7 +289,7 @@ func deleteReadyFile() {
 }
 
 func addCrdInformerEventHandlers(crdInformer cache.SharedIndexInformer, extclient extclientset.Interface, mgr manager.Manager, log logr.Logger) {
-	crdInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+	crdInformer.AddEventHandlerWithResyncPeriod(cache.ResourceEventHandlerFuncs{
 		AddFunc: func(obj interface{}) {
```

Comment:
Should handle `UpdateFunc` as well as `AddFunc` as I believe `UpdateFunc` is called on resync